app/experimenter/docs/adrs/0002-use-react-bootstrap-cra-gql-for-nimbus.md

```diff
@@ -0,0 +1,113 @@
+# Use React, Bootstrap, CRA, and GQL for the Nimbus Front-End Stack
+
+* Status: proposed
+* Deciders: Lauren Zugai, Jody Heavener, [TBD]
+* Date: 2020-09-08
+
+## Context and Problem Statement
+
+The front-end for Experimenter is currently composed of two applications. A more mature application for Normandy experiments, found at `app/experimenter/static/core`, is composed of a mix of Django templates, vanilla JS, jQuery, and React components. A newer React application for Nimbus experiments (previously named Rapid experiments) can be found at `app/experimenter/static/rapid`.
+
+With the decision made to build the Nimbus console as part of the Experimenter codebase ([see ADR #0001](https://github.com/mozilla/experimenter/blob/main/app/experimenter/docs/adrs/0001-build-nimbus-console-in-experimenter.md)), it was determined we likely want a "new front-end." This ADR, while slightly atypical as it does not serve to document a single architectural change, outlines a proposed directory tree change regarding the front-end, a proposed new front-end stack, how it compares to the current `rapid` stack, and how we will benefit from these changes.
+
+## Decision Drivers
+
+Drivers for directory tree changes:
+* A more obvious separation of concerns for:
+  * the front-end and back-end
+  * legacy front-end code and Nimbus front-end code
```

Comment:
Perhaps a naive question, but what are our intentions with legacy front-end Nimbus code? Assuming it is no longer needed with the new UI, does it need to stay in the repo or can it be purged [once the new UI is fully built, tested, deployed etc]?

Reply:
I'm thinking it'll be deleted since it won't be used.

Reply:
Once nimbus can replace all experimentation that goes through the old UI, it can be removed. We will need to make the old data available somehow either through migrating it or having some kind of statically accessible archive.

Reply:
Worth mentioning this in this ADR, or possibly in the pending [README](https://github.com/mozilla/experimenter/pull/3392#discussion_r485043805)?